src/main/resources/alma/common/subjects.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rules xmlns="http://www.culturegraph.org/metamorph">
+
+
+  <entity name="subject[]" flushWith="record">
+    <!-- RWSK Schlagwortfolgen fka: Schlagwortketten 1 - 10 -->
```

Comment:
*R**SW**K* = *R*egeln für die ***S***chlag***w***ort*k*atalogisierung :)

Reply:
Thanks.

Reply:
There are more of these... (`maps/rwsk-indicator.xml` and therein)

Reply:
Thanks, I thought I had all before I pushed at first. Will have another look.

Reply:
Should be fixed now.